Index: .idea/data_source_mapping.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"DataSourcePerFileMappings\">\r\n    <file url=\"file://$APPLICATION_CONFIG_DIR$/consoles/db/b94a448d-3c8e-4c2d-a310-637d14e93402/console.sql\" value=\"b94a448d-3c8e-4c2d-a310-637d14e93402\" />\r\n    <file url=\"file://$APPLICATION_CONFIG_DIR$/consoles/db/f308d5f0-d83b-4419-9ac3-aca90f9df1b7/console.sql\" value=\"f308d5f0-d83b-4419-9ac3-aca90f9df1b7\" />\r\n  </component>\r\n</project>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/data_source_mapping.xml b/.idea/data_source_mapping.xml
--- a/.idea/data_source_mapping.xml	(revision 08ad34c9f297ecaf2facf6ee3934875ff78b4b7c)
+++ b/.idea/data_source_mapping.xml	(date 1758732493584)
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="DataSourcePerFileMappings">
-    <file url="file://$APPLICATION_CONFIG_DIR$/consoles/db/b94a448d-3c8e-4c2d-a310-637d14e93402/console.sql" value="b94a448d-3c8e-4c2d-a310-637d14e93402" />
     <file url="file://$APPLICATION_CONFIG_DIR$/consoles/db/f308d5f0-d83b-4419-9ac3-aca90f9df1b7/console.sql" value="f308d5f0-d83b-4419-9ac3-aca90f9df1b7" />
   </component>
-</project>
+</project>
\ No newline at end of file
